views/question.pug

```diff
@@ -43,8 +43,10 @@ block content
             h4.sidebar-item#topic #{question.topic}
           .card-action
             h5.sidebar-item#points #{question.points} Points
-            p.sidebar-item#attempts This question has been attempted #{question.totalCount} times.
-            if question.answeredCount == 0
+            span.sidebar-item This question has been attempted&nbsp
+            span.sidebar-item#attempts #{question.totalAttemptsCount}
+            span.sidebar-item &nbsptimes.
+            if question.correctAttemptsCount == 0
```

Comment:
triple equal

Reply:
this is a pug file, pug does not have triple equal

Reply:
pug has triple equal, we use it in other files and its working

Reply:
we never do.....

Reply:
checkout home-sidebar.pug

Reply:
This line is not something im updating lol this was because of your pr, lemme pull

Reply:
yes that was my fault, please correct it

Reply:
no worries